2025-06-24 17:28:56,601 INFO    StreamThr :84760 [internal.py:wandb_internal():88] W&B internal server running at pid: 84760, started at: 2025-06-24 17:28:56.601319
2025-06-24 17:28:56,602 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: status
2025-06-24 17:28:56,603 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: status
2025-06-24 17:28:56,604 DEBUG   SenderThread:84760 [sender.py:send():303] send: header
2025-06-24 17:28:56,604 INFO    WriterThread:84760 [datastore.py:open_for_write():75] open: /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/run-fsrtnzbo.wandb
2025-06-24 17:28:56,622 DEBUG   SenderThread:84760 [sender.py:send():303] send: run
2025-06-24 17:29:01,624 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:06,626 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:11,628 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:16,630 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:21,631 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:26,633 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:31,635 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:36,637 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:41,639 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:46,641 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:50,338 ERROR   SenderThread:84760 [retry.py:__call__():154] Retry attempt failed:
Traceback (most recent call last):
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 466, in _make_request
    self._validate_conn(conn)
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1095, in _validate_conn
    conn.connect()
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connection.py", line 730, in connect
    sock_and_verified = _ssl_wrap_socket_and_match_hostname(
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connection.py", line 909, in _ssl_wrap_socket_and_match_hostname
    ssl_sock = ssl_wrap_socket(
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 469, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(sock, context, tls_in_tls, server_hostname)
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 513, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/ssl.py", line 1073, in _create
    self.do_handshake()
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/ssl.py", line 1342, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1132: The handshake operation timed out

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 843, in urlopen
    retries = retries.increment(
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/util/retry.py", line 474, in increment
    raise reraise(type(error), error, _stacktrace)
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/util/util.py", line 39, in reraise
    raise value
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 789, in urlopen
    response = self._make_request(
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 490, in _make_request
    raise new_e
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 468, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/home/yongxuan/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 369, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=10)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yongxuan/.local/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 113, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/yongxuan/.local/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 210, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/yongxuan/.local/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/yongxuan/.local/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/yongxuan/.local/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/transport/requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/site-packages/requests/api.py", line 115, in post
    return request("post", url, data=data, json=json, **kwargs)
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/yongxuan/anaconda3/envs/clip/lib/python3.8/site-packages/requests/adapters.py", line 713, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=10)
2025-06-24 17:29:51,643 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:29:57,661 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: poll_exit
2025-06-24 17:30:02,663 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:30:07,666 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:30:12,669 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:30:17,671 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:30:22,673 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:30:27,676 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:30:31,789 INFO    SenderThread:84760 [dir_watcher.py:__init__():216] watching files in: /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/files
2025-06-24 17:30:31,789 INFO    SenderThread:84760 [sender.py:_start_run_threads():928] run started: fsrtnzbo with start time 1750757336.600065
2025-06-24 17:30:31,789 DEBUG   SenderThread:84760 [sender.py:send():303] send: exit
2025-06-24 17:30:31,789 INFO    SenderThread:84760 [sender.py:send_exit():442] handling exit code: 1
2025-06-24 17:30:31,789 INFO    SenderThread:84760 [sender.py:send_exit():444] handling runtime: 0
2025-06-24 17:30:31,789 INFO    SenderThread:84760 [sender.py:_save_file():1171] saving file wandb-summary.json with policy end
2025-06-24 17:30:31,789 INFO    SenderThread:84760 [sender.py:send_exit():450] send defer
2025-06-24 17:30:31,789 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: poll_exit
2025-06-24 17:30:31,789 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:31,790 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 0
2025-06-24 17:30:31,790 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 0
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:transition_state():463] send defer: 1
2025-06-24 17:30:31,790 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:31,790 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 1
2025-06-24 17:30:31,790 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 1
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:transition_state():463] send defer: 2
2025-06-24 17:30:31,790 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:31,790 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 2
2025-06-24 17:30:31,790 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 2
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:transition_state():463] send defer: 3
2025-06-24 17:30:31,790 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:31,790 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 3
2025-06-24 17:30:31,790 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 3
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:transition_state():463] send defer: 4
2025-06-24 17:30:31,790 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:31,790 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 4
2025-06-24 17:30:31,790 DEBUG   SenderThread:84760 [sender.py:send():303] send: summary
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:_save_file():1171] saving file wandb-summary.json with policy end
2025-06-24 17:30:31,790 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:31,790 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 4
2025-06-24 17:30:31,791 INFO    SenderThread:84760 [sender.py:transition_state():463] send defer: 5
2025-06-24 17:30:31,791 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:31,791 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 5
2025-06-24 17:30:31,791 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:31,791 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 5
2025-06-24 17:30:31,791 INFO    SenderThread:84760 [sender.py:transition_state():463] send defer: 6
2025-06-24 17:30:31,791 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:31,791 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 6
2025-06-24 17:30:31,791 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:31,791 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 6
2025-06-24 17:30:31,791 INFO    SenderThread:84760 [sender.py:transition_state():463] send defer: 7
2025-06-24 17:30:31,791 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:31,791 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 7
2025-06-24 17:30:31,791 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:31,791 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 7
2025-06-24 17:30:31,791 INFO    SenderThread:84760 [dir_watcher.py:finish():362] shutting down directory watcher
2025-06-24 17:30:32,678 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: keepalive
2025-06-24 17:30:32,789 INFO    Thread-12 :84760 [dir_watcher.py:_on_file_created():275] file/dir created: /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/files/wandb-summary.json
2025-06-24 17:30:32,790 INFO    SenderThread:84760 [dir_watcher.py:finish():392] scan: /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/files
2025-06-24 17:30:32,790 INFO    SenderThread:84760 [dir_watcher.py:finish():406] scan save: /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/files/wandb-summary.json wandb-summary.json
2025-06-24 17:30:32,790 INFO    SenderThread:84760 [dir_watcher.py:finish():406] scan save: /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/files/config.yaml config.yaml
2025-06-24 17:30:32,790 INFO    SenderThread:84760 [sender.py:transition_state():463] send defer: 8
2025-06-24 17:30:32,790 DEBUG   HandlerThread:84760 [handler.py:handle_request():139] handle_request: defer
2025-06-24 17:30:32,791 INFO    HandlerThread:84760 [handler.py:handle_request_defer():162] handle defer: 8
2025-06-24 17:30:32,792 DEBUG   SenderThread:84760 [sender.py:send_request():317] send_request: defer
2025-06-24 17:30:32,792 INFO    SenderThread:84760 [sender.py:send_request_defer():459] handle sender defer: 8
2025-06-24 17:30:32,792 INFO    SenderThread:84760 [file_pusher.py:finish():168] shutting down file pusher
2025-06-24 17:30:33,713 WARNING StreamThr :84760 [internal.py:is_dead():385] Internal process exiting, parent pid 84696 disappeared
2025-06-24 17:30:33,713 ERROR   StreamThr :84760 [internal.py:wandb_internal():147] Internal process shutdown.
2025-06-24 17:30:33,792 INFO    WriterThread:84760 [datastore.py:close():279] close: /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/run-fsrtnzbo.wandb
2025-06-24 17:30:33,792 INFO    SenderThread:84760 [sender.py:finish():1331] shutting down sender
2025-06-24 17:30:33,792 INFO    SenderThread:84760 [file_pusher.py:finish():168] shutting down file pusher
2025-06-24 17:30:33,792 INFO    SenderThread:84760 [file_pusher.py:join():173] waiting for file pusher
2025-06-24 17:30:33,793 INFO    HandlerThread:84760 [handler.py:finish():814] shutting down handler
2025-06-24 17:30:35,404 INFO    Thread-15 :84760 [upload_job.py:push():143] Uploaded file /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/files/config.yaml
2025-06-24 17:30:35,404 INFO    Thread-14 :84760 [upload_job.py:push():143] Uploaded file /home/yongxuan/SurgVLP/wandb/run-20250624_172856-fsrtnzbo/files/wandb-summary.json
2025-06-24 17:30:35,604 INFO    Thread-11 :84760 [sender.py:transition_state():463] send defer: 9
